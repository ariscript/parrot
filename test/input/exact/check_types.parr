check: 
    val foo = (1,),
    val bar = (1,),
    val baz = (),
    foo is foo,
    foo is== foo,
    bar is bar,
    bar is== bar,
    foo is== bar,
    foo is bar,
    foo isn't baz,
    foo isn't== bar.

nil
