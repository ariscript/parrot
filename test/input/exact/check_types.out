failed test at 9:4-9:16: the left and right sides were not `==` ((1,) and (1,))
check block: 7/8 tests passed
nil
